Modules/_asynciomodule.c

```diff
@@ -2963,8 +2963,15 @@ task_step_handle_result_impl(asyncio_state *state, TaskObj *task, PyObject *resu
         if (task->task_must_cancel) {
             PyObject *r;
             int is_true;
+
+            // Beware: task->task_cancel_msg may be mutated and deleted
+            // prematurely in an evil `__getattribute__` function before being sent to `cancel`
+            // See https://github.com/python/cpython/issues/126138
+            PyObject* task_cancel_msg = Py_NewRef(task->task_cancel_msg);
             r = PyObject_CallMethodOneArg(result, &_Py_ID(cancel),
-                                             task->task_cancel_msg);
+                                task_cancel_msg);
```

Comment:
Can you align `task_cancel_msg` with `result` (or see if the function call fits on 1 line of 80-chars max)?

Reply:
If the limit is 80 not counting the whitespace before then it fits. Will adjust.

Reply:
The limit concerns the entire line (I think the file aligned pretty much on that limit). Otherwise, just align it on `result` and that'd be fine.

Reply:
I put it all on the same line and the linter didn't complain (not sure if it checks for that or not). I'll revert if it's an issue.

Reply:
We don't have a linter for that, it's just to be a bit consistent with PEP-7 as much as we can.

Reply:
Whitespace included the line is 84 characters long, I'll revert.